modules/ROOT/pages/getting-started-configuring.adoc

```diff
@@ -75,12 +83,32 @@ For an explanation of these terms, see link:https://www.keycloak.org/documentati
 
 :service-name: Identity Management
 
-[[choose-schema]]
-=== Choose the schema of a redirect url
+=== Configuring {idm-name}
+
+. Log into the {mobile-developer-console} and navigate to the {mobile-client} screen.
+
+. Select the *Mobile Services* tab.
+
+. If a binding to the {keycloak-service} service is in progress, a spinning icon is displayed to the right of the {keycloak-service} entry. Wait for the binding process to complete.
```

Comment:
Combine this step and the step above into one step:

". Select the *Mobile Services* tab and if a spinning icon is displayed to the right of the {keycloak-service} entry, wait for the binding process to complete."

Reply:
Should be fixed now.